RAW/PostModel.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  RawModel.swift
+//  RAW
+//
+//  Created by Bharat malhotra on 13/07/18.
+//  Copyright © 2018 Bharat malhotra. All rights reserved.
+//
+
+import Foundation
+
+struct PostModel: Codable {
+    let userid : Int?
```

Comment:
Name is Post. It's simpler and describes what it needs to

Reply:
done